### PR TITLE
Adding tun0 ip address to the segmants bar when the ip address is available.

### PR DESCRIPTION
**Pull request by CrunchV1**
It's quite nice when working with for example hackthebox or tryhackme.

### CODE REVIEW

File: themes/default.sh [hunk 1]
```diff
@@ -70,6 +70,7 @@ if [ -z $TMUX_POWERLINE_RIGHT_STATUS_SEGMENTS ]; then
 	TMUX_POWERLINE_RIGHT_STATUS_SEGMENTS=(
 		#"earthquake 3 0" \
 		"pwd 89 211" \
+		"tun0 13 255" \
```
Comment (erikw) [inline themes/default.sh:73]: As it's not common to have a tun interface, let's not add this to the default theme. It's easy to make your own theme outside that includes it :)

**Review by erikw — CHANGES_REQUESTED** [review]
Thanks for the PR. Just one thing to fix :)

File: themes/default.sh (hunk 1 above)
Comment (CrunchV1) [reply themes/default.sh:73]: Gotcha! Thanks for you time!

**Comment by erikw** [comment]
@CrunchV1 let's get this PR closed! It's just a small change to fix :)

**Comment by erikw** [comment]
@CrunchV1 Ah sorry I meant closed as it merged!!

**Pull request merged**